### PR TITLE
Update README.md to align w/ our current practices and a future CIRG …

### PR DESCRIPTION
**Pull request by mcjustin**
…Questionnaire profile

### CODE REVIEW

**Comment by mcjustin** [comment]
@achen2401  please look for your name inline, some questions for you there.

**Comment by mcjustin** [comment]
@achen2401 I've just edited this per our conversation today re `description`. There are a few more questions for you inline here, but not a big rush.

File: README.md [hunk 1]
```diff
@@ -1,39 +1,82 @@
...
+      - Examples of both in [MINICOG in the questionnaire filler](https://github.com/uwcirg/asbi-screening-app/blob/master/src/fhir/1_Questionnaire-MINICOG.json).
+    - "url": "http://hl7.org/fhir/StructureDefinition/rendering-xhtml"
+    - "valueString": XTHML
+    - Examples
+      - CIRG-PC-PTSD-5.json
+      - CIRG-PHQ-4.json
+      - 1_Questionnaire-USAUDIT.json (screener app)
+  - item[n]."extension"."valueCoding"
+    - We sometimes use this to indicate that an item is a score [here](https://github.com/uwcirg/fhir-questionnaires/pull/2/files#diff-66fd6a93556a044e8ffa3a290dac3e49b37b29b60c0cdddfb2645fe5cea49ae2R582)
+    - Amy do you read this for anything?
```
Comment (achen2401) [inline README.md]: @mcjustin I searched the repos we have, I don't see this being read for anything yet.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,82 @@
...
+  - item[n]."enableWhen"
+    - Amy you're not using this for anything, are you? I see it for the DCW Audit questionnaires, but we don't use those...
```
Comment (achen2401) [inline README.md]: @mcjustin this is used by ASBI questionnaire-to-survey tool to determine whether to show a question, see [here](https://github.com/asbi-cds-tools/questionnaire-to-survey/blob/63f117fd666830f4fdecec10ad4cea363abed413/fhirConversionTools.js#L346)

File: README.md [hunk 3]
```diff
@@ -1,39 +1,82 @@
...
+   - Contrast this with item[0]._text.extension (see below).
 - "code"
    - usage: uniquely identify the questionnaire according to some system, eg at LOINC. Not our canonical reference (see "id"). See https://github.com/uwcirg/fhir-questionnaires/pull/2/files#r974579864
 - "name"
-   - usage: arbitrary?
-- "linkId"
-   - usage: QuestionnaireResponse will refer to this
-   - REQUIRED.
+   - Standard says 0..1; computer friendly.
+   - Do we use this anywhere Amy? I see that it's populated for the DCW Questionnaires.
```
Comment (achen2401) [inline README.md]: @mcjustin `name` element, along with the `id` element is used by our ASBI app to determine what questionnaire(s) to query for.  See [here](https://github.com/uwcirg/asbi-screening-app/blob/9b4d6daa9b31bc799af1c69de612f8b495fd3e44/src/util/screening-selector.js#L211)

File: README.md (hunk 1 above)
Comment (mcjustin) [reply README.md]: thanks, I've just edited to reflect this.

File: README.md (hunk 2 above)
Comment (mcjustin) [reply README.md]: thanks, I've just edited to reflect this.

File: README.md (hunk 3 above)
Comment (mcjustin) [reply README.md]: thanks, I've just edited to reflect this.

**Pull request merged**